docs/modules/configuration/pages/jet-configuration.adoc

```diff
@@ -101,39 +101,36 @@ By default, resource uploading for Jet engine jobs is disabled.
 
 == List of Configuration Options
 
-The Hazelcast full example configuration files (https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/resources/hazelcast-full-example.yaml#L3509[YAML],
-https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/resources/hazelcast-full-example.xml#L3612[XML]) in your distribution
-includes a description of all configuration options.
-Jet engine configuration options are listed below:
+The _hazelcast-full-example_ configuration files (YAML, XML) available in your distribution and linked from xref:configuration:configuring-declaratively.adoc[] include a description of all configuration options. The configuration options for Jet engine are also listed below:
```

Comment:
can we clarify that it isn't that both YAML and XML config files need to be updated?

Reply:
Not sure what you mean here - the files are included just for reference. For context I changed the hardcoded link to these files to point to the page instead - are you meaning we should provide more context on how these files are used?

Reply:
Op, took your suggestion - think that clarifies